src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -63,5 +65,6 @@ public PersonCard(Person person, int displayedIndex) {
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
         birthday.setText(Optional.ofNullable(person.getBirthday()).orElse(new Birthday("")).toString());
+        moneyOwed.setText(person.getMoneyOwed().toString());
```

Comment:
Here, we can display some meaningful text instead of `10.35` or `-3.20`, which might not be very intuitive as to what it is.

Reply:
Made necessary changes